.claude/skills/land-stacked-prs/sf_wait_green.sh

```
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+# Wait until every check on the given PRs has completed (or 25 min), then print a merge-readiness summary.
+R="${SF_REPO:-rmems/synthetic-factory}"; deadline=$((SECONDS+1500))
+for n in "$@"; do
+  while :; do
+    j=$(gh pr view $n --repo $R --json state,mergeable,mergeStateStatus,headRefOid,statusCheckRollup 2>/dev/null) || { sleep 20; continue; }
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)